.github/workflows/release.yml

```diff
@@ -225,11 +225,39 @@ jobs:
           process_dependencies $target $destDir $APP_PREFIX/$target/SpaceAcres.app/Contents/MacOS/space-acres
         if: runner.os == 'macOS'
 
+      - name: Sign Application (macOS)
```

Comment:
I don't think we want to sign individual files, instead we probably want to sign dmg file, which needs to be created first.

Also we had to zip files on macOS for signatures to be preserved or else executable downloaded from GitHub will not have a signature attached to it because signature for executables is stored as file system-level metadata. Not sure if that is the case for dmg files though or dmg images store signatures in a different way that is not succeptible to this.